contracts/paymaster/src/forward_call.rs

```diff
@@ -0,0 +1,61 @@
+multiversx_sc::imports!();
+
+pub type PaymentsVec<M> = ManagedVec<M, EsdtTokenPayment<M>>;
+
+static ERR_CALLBACK_MSG: &[u8] = b"Error received in callback:";
+
+#[multiversx_sc::module]
+pub trait ForwardCall {
+    fn forward_call(
+        &self,
+        dest: ManagedAddress,
+        endpoint_name: ManagedBuffer,
+        endpoint_args: MultiValueEncoded<ManagedBuffer>,
+        payments: PaymentsVec<Self::Api>,
+    ) {
+        let original_caller = self.blockchain().get_caller();
+
+        if !self.blockchain().is_smart_contract(&dest) {
+            self.send().direct_multi(&dest, &payments);
```

Comment:
endpoint and arguments - add the message as well in the direct send.

Reply:
Created a contract_call for that.
`direct_multi` doesn't support messages anymore.